code/modules/mob/living/carbon/alien/alien.dm

```diff
@@ -7,7 +7,7 @@
 	name = "alien"
 	voice_name = "alien"
 	icon = 'icons/mob/alien.dmi'
-	gender = NEUTER
+	gender = FEMALE //All xenos are girls!!

```

Comment:
Well, that is the intent, however it does not, because examine code is trash. Take a look at how humans and carbons are done.